packages/form/src/assets/css/form.css

```diff
@@ -74,3 +74,7 @@ form span[role="alert"] {
   color: var(--color-alert-danger, red);
   font-size: var(--form-input-error-font-size, 0.8em);
 }
+
+.field .p-dropdown {
+  width: 100%;
+}
```

Comment:
You can remove this style from here. It is already added above as `form .field .p-dropdown`